src/components/input/Input.jsx

```diff
@@ -0,0 +1,18 @@
+function Input({ type, name, placeholder }) {
+  return (
+    <>
+      <input
+        type={type}
+        name={name}
+        placeholder={placeholder}
+        className=' my-2 block w-full rounded-lg
```

Comment:
function Input({ type, name, placeholder, className }) {
  return (
    <>
      <input
        type={type}
        name={name}
        placeholder={placeholder}
        // here is the added dynamic code line!
        className={className}
        required
      />
    </>
  )
}

export default Input